src/TestTask/components/Header/Header.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react'
```

Comment:
I wonder why did you create a whole new header here? It wasn't part of the task.

Reply:
Oh, I see now, you've created whole new directory for this task. It was unncecessary, subdirectory in 'components' for component from mockups would be enough

Reply:
there is too much redundant, copied code

Reply:
I noticed that components from the mockup were similar to ones in the project, some of them were even identical. However having read the task instruction carefully, I realised, that I was expected to create components rather than use available ones. This was also the reason of creating a whole new directory for the task.